message_utils.py

```
@@ -1,8 +1,33 @@
 import discord
 import re
-from typing import Optional, Dict, Any
+from typing import Optional, Dict, Any, List
```

Ruff (0.11.9)

`typing.List` imported but unused

Remove unused import: `typing.List`

(F401)